index.d.ts

```diff
@@ -200,10 +278,16 @@ declare namespace RNTrackPlayer {
   export const CAPABILITY_DISLIKE: Capability;
   export const CAPABILITY_BOOKMARK: Capability;
 
-  export const PITCH_ALGORITHM_LINEAR: PitchAlgorithm;
-  export const PITCH_ALGORITHM_MUSIC: PitchAlgorithm;
-  export const PITCH_ALGORITHM_VOICE: PitchAlgorithm;
-  
+  export const PITCH_ALGORITHM_LINEAR: typeof PITCH_ALGORITHM_LINEAR_IOS | typeof PITCH_ALGORITHM_ANDROID;
+  export const PITCH_ALGORITHM_MUSIC: typeof PITCH_ALGORITHM_MUSIC_IOS | typeof PITCH_ALGORITHM_ANDROID;
+  export const PITCH_ALGORITHM_VOICE: typeof PITCH_ALGORITHM_VOICE_IOS | typeof PITCH_ALGORITHM_ANDROID;
+
+  export const PITCH_ALGORITHM_LINEAR_IOS: "linear";
+  export const PITCH_ALGORITHM_MUSIC_IOS: "music";
+  export const PITCH_ALGORITHM_VOICE_IOS: "voice";
+
+  export const PITCH_ALGORITHM_ANDROID: undefined;
```

Comment:
These other string constants are only defined in JS, which shouldn't be different between platforms, even if they are not supported. I feel like this is not the proper way to represent constants that are not supported in a specific platform.